src/components/Header.tsx

```diff
@@ -1,41 +1,95 @@
-import React from 'react'
-import AppBar from '@mui/material/AppBar';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import MuiAppBar from '@mui/material/AppBar';
 import Box from '@mui/material/Box';
 import Toolbar from '@mui/material/Toolbar';
-import Typography from '@mui/material/Typography';
-import IconButton from '@mui/material/IconButton';
+import MuiIconButton from '@mui/material/IconButton';
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+
 import MenuIcon from '@mui/icons-material/Menu';
-import { Button } from '@mui/material';
-interface Props {
-  window?: () => Window;
-}
-const navItems = ['Home', 'About', 'Contact'];
+import SettingsIcon from '@mui/icons-material/Settings';
+import NotificationsNoneIcon from '@mui/icons-material/NotificationsNone';
+import AccountCircle from '@mui/icons-material/AccountCircle';
 
-const Header = (props: Props) => {
-  const { window } = props;
-  const [mobileOpen, setMobileOpen] = React.useState(false);
+interface HeaderProps {
+  toggleDrawer: () => void;
+}
+const Header = ({ toggleDrawer }: HeaderProps) => {
+  const [isAnchor, setIsAnchor] = useState(false);
 
-  const handleDrawerToggle = () => {
-    setMobileOpen(!mobileOpen);
+  const handleAnchorMenu = () => {
+    setIsAnchor(!isAnchor);
```

Comment:
```suggestion
    setIsAnchor(prev => !prev);
```
이와 같이 값을 직접 바꾸기보다 이전 상태를 바꿔주는 건 어떨까요? (바꾼다면 prev 이름은 더 적절하게 하면 좋을 것 같아요!)